populate/src/main/resources/seed/portals/ourhealth/studies/ourheart/surveys/ourHealthConsent.json

```diff
@@ -10,12 +10,13 @@
   "calculatedValues": [
     {
       "name": "oh_oh_consent_signatureDate",
-      "expression": "currentDate()",
+      "expression": "iif({oh_oh_consent_signatureDate}, {oh_oh_consent_signatureDate}, currentDate())",
       "includeIntoResult": true
     },
     {
-      "name": "formattedSignatureDate",
-      "expression": "formatDate({oh_oh_consent_signatureDate})"
+      "name": "oh_oh_consent_formattedSignatureDate",
+      "expression": "formatDate({oh_oh_consent_signatureDate})",
```

Comment:
Dumb question, but what happens when `formatDate` returns null?
Is that just an empty string?